lib/i18n/app_en.json

```diff
@@ -0,0 +1,5 @@
+{
+  "common": {
+    "appName": "Green App"
+  }
+}
```

Comment:
This is only meant as an example obviously. I think I'd not keep the English translation file up to date (or even delete it for now), as a fallback still German will be used.
We can then at some point request translations from the BGS.